src/components/ExpensiTextInput/propTypes.js

```diff
@@ -33,7 +33,7 @@ const defaultProps = {
     label: '',
     errorText: '',
     placeholder: '',
-    error: false,
+    hasError: false,
```

Comment:
I don't see any existing usages of this prop being updated. Was this prop not in use, or do we need to update existing usages?

Reply:
I think it wasn't being used. It was declared as `hasError` in the `propTypes`, so I just corrected that misalignment  of names

Reply:
While we are here should we rename this file to `expensiTextInputPropTypes.js`? 

I noticed that the convention of using just `propTypes.js` is not correct (and all other propTypes files reference what they are propTypes for).

Reply:
> While we are here should we rename this file to `expensiTextInputPropTypes.js`?
> 
> I noticed that the convention of using just `propTypes.js` is not correct (and all other propTypes files reference what they are propTypes for).

Ok, but I couldn't find in our style guidelines something about the naming of this file. In our codebase I can see it named starting with uppercase and starting with lowercase, which one do we prefer? I'm not sure because... I would lean toward uppercase just because components filename start with uppercase, but then, the propType is not a component itself, so it should be lowercase?

Examples:

- PickerPropTypes
- plaidLinkPropTypes
- inlineCodeBlockPropTypes
- onfidoPropTypes
- HoverablePropTypes

Reply:
I'm not sure if it's in the style guide but in my opinion 

>propType is not a component itself, so it should be lowercase 

is the correct answer. Pascal case is usually reserved for components, classes, namespaces etc.